marmoset/core/marmosetbrowser.py

```diff
@@ -46,7 +50,7 @@ class Marmoset():
     @ivar base_url: The base url for the marmoset submission server.
     """
     base_url = "http://marmoset.student.cs.uwaterloo.ca"
-    cookiefile = "/tmp/marmoset.session.cookies"
+    cookiefile = tempfile.gettempdir() + '/marmoset.session.cookies'
```

Comment:
Can you use `os.path.join` here and remove the `/` prefix from `/marmoset.session.cookies`.  Better for cross-platform compatibility.